test/rubygems/test_gem_specification.rb

```diff
@@ -2781,6 +2781,14 @@ def test_validate_description
       end
 
       assert_equal %("#{f}" or "#{t}" is not a description), e.message
+
+      # Allow these words anywhere after the start of the description.
+      # See https://github.com/rubygems/rubygems/issues/4808
+      @a1.description = "(some description) #{f}"
```

Comment:
Don't we want to check if `TODO` or `FIXME` is in this string?

Reply:
Yes, and FIXME _is_ in that string -- that's what `#{f}` is. The words TODO/FIXME are obfuscated throughout this entire part of the codebase, presumably so `git grep -E 'TODO|FIXME'` to find areas that need work doesn't give false positives.

Reply:
See this part of the file: https://github.com/rubygems/rubygems/blob/0c50fd5c4ad7da55affece14d8307ec8aba78942/test/rubygems/test_gem_specification.rb#L2551-L2554

Reply:
I think you also want to check the same thing with `#{t}` as well to cover the change in the corresponding regexp?

As a personal nitpick, I usually avoid linking to specific GitHub issues. If something needs a code comment, I'd rather explain things from scratch. Some reasons why I prefer that:

* A short explanation is usually about the same length as a GitHub link.
* It makes the code independent from where it's hosted. For example, when we moved the repository, we had to update hundreds of code links like this.
* I think a link might actually distract the reader and make her lose focus.
* If more information is really needed, the PR that introduced the code can be found through `git blame` or other tools easily.
* If we added a link to the closed issue every time we fixed an issue, the source code would be full of those links, and if we do it only in some cases, it's hard to explain why those cases are special.

It's not a strong opinion anyways, and I'm happy with leaving it if you think it's best to link to the full issue in this case!

Reply:
I'll add a check with `#{t}` and replace the link with a better description.

Reply:
Done. :tada: